middleware/allRoutes/handler.go

```diff
@@ -20,6 +20,14 @@ func Handler(routesHandler map[string]http.Handler) func(h http.Handler) http.Ha
 
 			// No point calling zebedee for these paths so skip middleware
 			if ok, err := regexp.MatchString(`^\/(datasets|filter|feedback|healthcheck).*$`, path); ok && err == nil {
+				log.Info("Skipping content specific handling as not relevant on this path.", log.Data{"url": path})
+				h.ServeHTTP(w, req)
+				return
+			}
+
+			// We can skip handling based on content type where the url points to a known/expected file extension
+			if ok, err := regexp.MatchString(`^*(xls|zip|csv|xslx)$`, req.URL.String()); ok && err == nil {
```

Comment:
might want `\.` before the group of extensions? this could match a page that just happens to end with one of those, e.g. `aboutxls`